tests/restore/dataset/test_dataset_restore_items_generator.py

```diff
@@ -196,15 +178,13 @@ def __prepare_tables_which_should_be_returned_by_max_partition_days_query():
             project_id=PROJECT_TO_RESTORE,
             dataset_id=DATASET_TO_RESTORE,
             table_id='tbl1',
-            date=datetime(2017, 12, 6),
-            partition_id='20171206'
```

Comment:
after change, your tests doesn't generate partitions in datastore.

In my opinion it should, we still want to test how our application works for partitioned table.